templates/sub-01/ses-01/eeg/sub-01_task-FullExample_acq-CTF_run-1_proc-sss_eeg.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
This file looks fine, except for the file name. Why is there still acq-CTF and prod_sss?